### DISCUSSION TITLE
Generating printable file for meeting directory.

### DISCUSSION THREAD

**Discussion by AASTL** [code4recovery/12-step-meeting-list#1473] [Q&A]
Hello!
I am the office manager for the AA central office in St. Louis. (aastl.org) Most of the content here is WAAAY beyond me, so thanks for reading my novice-level question! We have a WordPress website, with the TSML plugin for our meeting directory.

I recall that at our annual intergroup/central office seminar, last fall in Cincinatti, we were told that the TSML plugin would soon include a way of generating a printable file for paper meeting directories. Has this project been completed?

I have a volunteer helping me to create a paper/printed directory, and I need to let her know if that project is complete, or still in the works? 

Thank you!

**Comment by thessnake03** [comment]
The only bad question is one that doesn't get asked.

You can export your current meetings as a pdf. In the WordPress dashboard, go to Meetings, then import/export. Click the 'Generate PDF' button.

![screenshot_2024-07-23_at_4.18.23___pm.png](https://github.com/user-attachments/assets/c73bbad6-fa8d-4a32-8bb1-40debef2dbb9)

Hope that helps
